Paranormal/ParanormalTests/DocumentTests.swift

```diff
@@ -63,9 +63,9 @@ class DocumentTests: QuickSpec {
                 }
 
                 it("sets brush color to ZUP"){
-                    expect(document.currentColor.redComponent).to(equal(0.5))
-                    expect(document.currentColor.greenComponent).to(equal(0.5))
-                    expect(document.currentColor.blueComponent).to(equal(1.0))
+                    expect(document.toolSettings.colorAsNSColor.redComponent).to(equal(0.5))
```

Comment:
Use the beColor matcher. expect(document.toolSettings.colorAsNSColor).to(beColor(0.5, 0.5, 1.0) should work.